lib/test.js

```diff
@@ -70,6 +70,9 @@ Object.keys(assert).forEach(function (el) {
 	};
 });
 
+Test.prototype._capt = assert._capt
+Test.prototype._expr = assert._expr
```

Comment:
Thanks! These methods capture each node value in assertion expressions then pass the values to power-assert formatter/renderer.